src/test_workflow/test_cluster.py

```diff
@@ -80,6 +82,9 @@ def start(self) -> None:
         for service in self.all_services:
             service.wait_for_service()
 
+        self.start_stdout = self.service.process_handler.stdout_data
```

Comment:
Should this be called `stdout_data` instead? Why is it `start`?

Reply:
thanks for the asking again..
I was thinking of using the argument names that can represent in which stage that the data was generated.
On L51, 52, I set the arguments to be start_stdout, and start_stderr as the outputs generated at the stage start().
Will you suggest changing to start_stdout_data and start_stderr_data? Maybe that will be less confusion? Thanks!

Reply:
Updated with start_stdout_data and start_stderr_data. thanks